sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2834,24 +2834,24 @@
         }
       },
       "CloudToDeviceMethod": {
-        "description": "Parameters to execute a direct method on the device",
+        "description": "Parameters to execute a direct method on the device.",
         "type": "object",
         "properties": {
           "methodName": {
-            "description": "Method to run",
+            "description": "Name of the method which is to be executed.",
             "type": "string"
           },
           "payload": {
-            "description": "Payload",
+            "description": "The JSON-formatted direct method payload, up to 128kb in size.",
             "type": "object"
           },
           "responseTimeoutInSeconds": {
-            "description": "Timeout for the method invocation to return a response",
+            "description": "Timeout for the method invocation to return a response, defaults to 30 seconds. Minimum allowed value is 5 seconds, maximum allowed value is 300 seconds.",
```

Comment:
Fullstop or semicolon after "return a response" instead of comma

Reply:
This is from an old commit Sindhu, can you review this instead: https://github.com/Azure/azure-sdk-for-net/commit/3264d2348b91a956b2e7a132c463d0940d85abe9